app/src/main/java/io/github/droidkaigi/confsched/util/DataBindingAttributeUtil.java

```diff
@@ -22,15 +22,16 @@
 
 public class DataBindingAttributeUtil {
 
-    @BindingAdapter("speakerImageUrl")
-    public static void setSpeakerImageUrl(ImageView imageView, @Nullable String imageUrl) {
+    @BindingAdapter({"speakerImageUrl", "speakerImageSize"})
+    public static void setSpeakerImageUrlWithSize(ImageView imageView, @Nullable String imageUrl, float sizeInDimen) {
         if (TextUtils.isEmpty(imageUrl)) {
             imageView.setImageDrawable(ContextCompat.getDrawable(imageView.getContext(), R.drawable.ic_speaker_placeholder));
         } else {
+            final int size = (int)(sizeInDimen + 0.5f);
```

Comment:
Maybe `imageView.getContext().getResources().getDimension(R.dimen.line_small)` is better, right?